p4-16/psa/PSA.mdk

```diff
@@ -2422,7 +2431,11 @@ In particular, ActionProfile and ActionSelector single operations,
 such as adding a member to a group, removing a member from a group,
 adding an empty group, deleting an empty group, or modifying the
 action parameters of an action added earlier to a group, should all be
-atomic.
+atomic.  One possible exception is a PSA implementation that uses the
```

Comment:
double whitespace again it seems (and there are other occurrences as well). This is funny because I have seen it in a PR from Tom Everman as well; does it come from the text editor?

Reply:
I checked Wikipedia on this, out of curiosity.  Quoting from the first paragraph of the Wikipedia page below: "Many people prefer double sentence spacing for informal use because that was how they were taught to type."  <-- That is my reason.

https://en.wikipedia.org/wiki/Sentence_spacing

Reply:
if you use emacs it has some very nice commands to operate on phrases.
phrases in emacs are delimited by a sentence terminator and two spaces.

Reply:
more precisely, that's the default regular expression for phrases. you can customize it. but it makes expressions such as i.e. or e.g. that and with a period unambiguous.

Reply:
@jafingerhut My bad I had no idea this was a thing. I should have Googled it first.

Reply:
@antoninbas Not bad on your part at all.  Until a month ago I did not even realize that a few people are vehemently anti-double-space-between-sentences.

Reply:
Lol. Given that the using a single space after a period is called "French spacing" (https://en.wikipedia.org/wiki/History_of_sentence_spacing#French_and_English_spacing), I think my ignorance can be excused :)